zhaquirks/sinope/switch.py

```diff
@@ -294,517 +372,460 @@ class SinopeTechnologiesMeteringCluster(CustomCluster, Metering):
     class AttributeDefs(Metering.AttributeDefs):
         """Sinope Manufacturer Metering Cluster Attributes."""
 
-        status: Final = foundation.ZCLAttributeDef(
+        status: Final = ZCLAttributeDef(
             id=0x0200, type=ValveStatus, access="r", is_manufacturer_specific=True
         )
-        unit_of_measure: Final = foundation.ZCLAttributeDef(
+        unit_of_measure: Final = ZCLAttributeDef(
             id=0x0300, type=UnitOfMeasure, access="r", is_manufacturer_specific=True
         )
 
 
-class SinopeTechnologiesFlowMeasurementCluster(CustomCluster, FlowMeasurement):
-    """Custom flow measurement cluster that divides value by 10."""
+(
+    # <SimpleDescriptor(endpoint=1, profile=260,
+    # device_type=81, device_version=0,
+    # input_clusters=[0, 3, 6, 1794, 2820, 65281]
+    # output_clusters=[25]>
+    # <SimpleDescriptor(endpoint=1, profile=260,
+    # device_type=81, device_version=0,
+    # input_clusters=[0, 3, 6, 1794, 2820, 4096, 65281]
+    # output_clusters=[25, 4096]>
+    QuirkBuilder(SINOPE, "SP2600ZB")
+    .applies_to(SINOPE, "SP2610ZB")
+    .replaces(SinopeTechnologiesMeteringCluster)
+    .replaces(SinopeManufacturerCluster)
+    .sensor(  # Current summ delivered
+        attribute_name=SinopeTechnologiesMeteringCluster.AttributeDefs.current_summ_delivered.name,
+        cluster_id=SinopeTechnologiesMeteringCluster.cluster_id,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        unit=UnitOfEnergy.WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY,
+        reporting_config=ReportingConfig(
+            min_interval=59, max_interval=1799, reportable_change=60
+        ),
+        translation_key="current_summ_delivered",
+        fallback_name="Current summ delivered",
+        entity_type=EntityType.STANDARD,
+    )
```

Comment:
Same for this. ZHA should already create an entity for this `Metering` cluster, so we shouldn't add a custom entity.
Does the ZHA entity not appear for some reason?